src/settings/Settings.js

```diff
@@ -158,6 +165,27 @@ export const SETTINGS = {
         displayName: _td("Show info about bridges in room settings"),
         default: false,
     },
+    "fontSize": {
+        displayName: _td("Font size"),
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        default: 16,
+        controller: new FontSizeController(),
+    },
+    "fontSizeMin": {
+        displayName: _td("Min font size"),
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        default: 13,
+    },
+    "fontSizeMax": {
+        displayName: _td("Max font size"),
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        default: 20,
+    },
```

Comment:
Github insists that I commented on an outdated commit:

It looks like these settings aren't something the user can change and are instead hardcoded to a particular value. A pair of constants somewhere in the code would be preferred to avoid abusing the settings system (and to avoid a situation when someone inevitably tries to set this to `min: 10, max: -10` with localstorage).

Reply:
I've made them constants under FontWatcher instead.